bundle/src/main/java/com/adobe/acs/commons/workflow/synthetic/impl/SyntheticMetaDataMap.java

```diff
@@ -29,8 +29,8 @@
 import java.util.Map;
 import java.util.Set;
 
-public class SyntheticMetaDataMap implements MetaDataMap {
-    private ValueMap metaDataMap;
+public class SyntheticMetaDataMap implements MetaDataMap, com.day.cq.workflow.metadata.MetaDataMap {
```

Comment:
This is the only place where the CQ class is fully qualified; everywhere else the Granite API is the fully qualified class.